### DISCUSSION TITLE
Standard C++ Library header files not found

### DISCUSSION THREAD

**Discussion by howz97** [clangd/clangd#2077] [Q&A]
My clangd does not work anymore suddenly. I can compile and run this code correctly.
![Screenshot from 2024-07-01 23-45-52](https://github.com/clangd/clangd/assets/33745049/9346e87f-db83-420b-85d3-660954af978c)

**Comment by HighCommander4** [comment]
Have a look at https://clangd.llvm.org/troubleshooting#cant-find-standard-library-headers-map-stdioh-etc

**Comment by howz97** [comment]
After reading documents with patience and inspect the output of `clangd --check=main.cpp`, i found this is because i have a g++11 and a not completely removed g++12 on my system.
